www/src/config.ts

```diff
@@ -344,6 +345,37 @@ export const SIDEBAR: Sidebar = {
       { text: "Netlify", link: "ru/deployment/netlify" },
     ],
   },
+  nl: {
+    "Create T3 App": [
+      { text: "Introduction", link: "nl/introduction" },
+      { text: "Waarom CT3A?", link: "nl/why" },
+      { text: "Installatie", link: "nl/installation" },
+      { text: "Mappen-structuur", link: "nl/folder-structure" },
+      { text: "FAQ", link: "nl/faq" },
+      { text: "T3 Collectie", link: "nl/t3-collection" },
+      { text: "Voorbeelden", link: "nl/examples" },
+      { text: "Andere Aanbevelingen", link: "nl/other-recs" },
+    ],
+    Usage: [
+      { text: "First Steps", link: "nl/usage/first-steps" },
```

Comment:
In [other languages](https://github.com/t3-oss/create-t3-app/pull/1807/files#diff-8bce9fffdecb43de16832eb9541eeaccf09bcdfe30f880d87e6a7b17a698bfd4R272), this title is translated in the language of the translation. So this would be: "Eerste Stappen".